### DISCUSSION TITLE
Ability to Filter MCA Dashboard by Destination Name in addition to Channel Name.

### DISCUSSION THREAD

**Discussion by JeffBenedict** [nextgenhealthcare/connect#5982] [Feature/Enhancement Ideas]
It would be nice if the Filter on the MCA Dashboard also found matching destination names, in addition to matching the channel name.  This would allow a better / more complete view of an entire deployed solution within Mirth.

**Comment by kirbykn2** [comment]
@JeffBenedict ,

You may want to use tags to get a more complete view of a solution.  We use tags for things like incoming/outgoing, vendor or partner name, connection type (poll, https, sftp, etc.), message types (ADT, ORU, DFT, etc).  You can then use the tags to find various combinations of channels.

**Comment by JeffBenedict** [comment]
@kirbykn2, we do have tags for our solutions, and getting them setup was a pain, especially when we have Inbound channels with 10-15 destinations; having to take screen shots and then enter that inbound channel to 10-15 application tags.  I was just stating that it would be nice to be able to search destinations too.
